AdminUI/README.md

```diff
@@ -10,12 +10,6 @@ By default, this creates a local SQL db that you can test on. To change this, al
 
 In the Package Manager Console, run the following commands:
```

Comment:
I may be missing something, but if we remove those commands does this line make sense?

Reply:
no you right, I fixed it